### PR TITLE
16 single card component

### PR DESCRIPTION
**Pull request by dazu1990**
Added single card component - demo video: 

https://github.com/user-attachments/assets/e82bb4c2-47b8-4195-857d-dece5fab320f

updated styling: 
Screenshot 2024-12-16 at 8 16 56 PM

Supports :

standard cards
double sided cards
cards with broken images
text search

TO-DO:
- clean up styling for mobile
- clean up styling for text 
- polish on button and general styling

### CODE REVIEW

File: resources/css/app.css [hunk 1]
```diff
@@ -1,3 +1,3 @@
 @tailwind base;
 @tailwind components;
-@tailwind utilities;
+@tailwind utilities;
```
Comment (dazu1990) [inline resources/css/app.css:3]:
```suggestion
@tailwind utilities;

```

**Comment by nickzou** [comment]
@dazu1990 did you have `composer run dev` while you wrote this?

**Comment by dazu1990** [comment]
@nickzou implemented your changes + some refactoring

File: resources/js/Pages/Dashboard.tsx [hunk 2]
```diff
@@ -36,6 +37,120 @@ export default function Dashboard() {
...
                     </div>
+                    {/* this is all dummy content for testing and should be deleted eventually */}
+                    <div className="flex w-full flex-wrap med:mx-[-2]">
+                        <div className="mb-2 w-full px-2 md:w-1/2 lg:w-1/4">
+                            <p className="text-center">Double sided card</p>
+                            <MTGCard
+                                imgSrc="https://cards.scryfall.io/large/front/1/8/18a2bdc8-b705-4eb5-b3a5-ff2e2ab8f312.jpg?1626663778"
+                                title="Wandering Archaic"
+                                cardSuperType="Creature"
+                                cardType='Avatar'
```
Comment (nickzou) [inline resources/js/Pages/Dashboard.tsx:48]: @dazu1990 should be double quote as per lint rules, suprised prettier didn't catch this.

**Review by nickzou — CHANGES_REQUESTED** [review]
@dazu1990 Just one crucial change. The `med` one. The other comments should be addressed but aren't breaking in anyway. Some once we have CI set up, it'll just fail it anyway.

File: resources/js/Pages/Dashboard.tsx (hunk 2 above)
Comment (dazu1990) [reply resources/js/Pages/Dashboard.tsx:48]: fixing manually - let's pair tonight to see why prettier isn't catching it

File: resources/js/Pages/Dashboard.tsx [hunk 3]
```diff
@@ -36,6 +37,120 @@ export default function Dashboard() {
...
+                                manaCost="{5}"
+                                description="Whenever an opponent casts an instant or sorcery spell, they may pay {2}. If they don’t, you may copy that spell. You may choose new targets for the copy."
+                                powerToughness={['4','4']}
```
[the remark this answers is not in the document]
Comment (dazu1990) [reply resources/js/Pages/Dashboard.tsx:51]: not seeing the squigglies - we'll review in person tonight

**Review by nickzou — APPROVED** [review]
lgtm

**Pull request merged**